### DISCUSSION TITLE
Dashboard log collection issue

### DISCUSSION THREAD

**Discussion by zh199609** [seleniumbase/SeleniumBase#2858] [General]
After using pytest-assume, assertion failure will not be reflected in the dashboard

**Comment by mdmintz** [comment]
Duplicate of https://github.com/seleniumbase/SeleniumBase/issues/2442#issuecomment-1902117227 and https://github.com/seleniumbase/SeleniumBase/discussions/1847#discussioncomment-5673272.

SeleniumBase comes with its own system for non-terminating checks (which allows for multiple failing assertions per test).
